src/main/java/mate/jdbc/util/ConnectionUtil.java

```diff
@@ -6,10 +6,10 @@
 import java.util.Properties;
 
 public class ConnectionUtil {
-    private static final String URL = "YOUR DATABASE URL";
-    private static final String USERNAME = "YOUR USERNAME";
-    private static final String PASSWORD = "YOUR PASSWORD";
-    private static final String JDBC_DRIVER = "YOUR DRIVER";
+    private static final String URL = "jdbc:mysql://localhost:3306/taxi_service";
+    private static final String USERNAME = "user";
+    private static final String PASSWORD = "password";
```

Comment:
Hardcoding credentials in the source code is a bad practice. It would be better to use configuration files or environment variables. This way, you can avoid accidents like committing sensitive data into a public repository.